sdk.d.ts

```diff
@@ -70,38 +70,38 @@ declare class PaddleSDK {
      * Get the current list of plans for a subscription
      *
      * @method
-     * @param {number} productID
+     * @param {number} [productID]
      * @returns {Promise}
      * @fulfil {object} - The plans list
      *
      * @example
      * const plans = await client.getProductPlans(123);
      */
-    getProductPlans(productID: number): Promise<any>;
+    getProductPlans(productID?: number): Promise<any>;
     /**
      * Get the current list of users for a subscription plan
      *
      * @method
-     * @param {number} planID
+     * @param {number} [planID]
      * @returns {Promise}
      * @fulfil {object} - The users list
      *
      * @example
      * const users = await client.getPlanUsers(123);
      */
-    getPlanUsers(planID: number): Promise<any>;
+    getPlanUsers(planID?: number): Promise<any>;
     /**
      * Get the list of payments for a subscription plan
      *
      * @method
-     * @param {number} planID
+     * @param {number} [planID]
```

Comment:
This needs updating in the JSDoc in the source file as well.

Reply:
Docs updated as requested